src/parameterStore.ts

```diff
@@ -19,23 +19,56 @@ function getSSM() {
     return new AWS.SSM();
 }
 
-// eslint-disable-next-line import/prefer-default-export
-export async function getIntegrationTransformUrl(): Promise<string> {
-    const ssm = getSSM();
-    let path = process.env.INTEGRATION_TRANSFORM_PATH;
+function getParamPath(paramEnvVarPath: string, defaultPath: string): string {
+    const path = process.env[paramEnvVarPath];
     if (path === undefined) {
-        throw new Error('INTEGRATION_TRANSFORM_PATH is not defined in environment variables');
+        throw new Error(`${paramEnvVarPath} is not defined in environment variables`);
     }
     if (IS_OFFLINE === 'true') {
-        path = `fhir-service.integration-transform.${AWS_REGION}.${STAGE}.url`;
+        return defaultPath;
     }
 
+    return path;
+}
+
+async function getParams(paramStorePaths: string[]): Promise<Record<string, string>> {
+    const ssm = getSSM();
+
     const data = await ssm
-        .getParameter({
-            Name: path,
+        .getParameters({
+            Names: paramStorePaths,
             WithDecryption: true,
         })
         .promise();
 
-    return data.Parameter!.Value!;
+    if (data.InvalidParameters === undefined || data.InvalidParameters.length > 0) {
+        throw new Error(`Unable to find these paths in AWS Param Store: ${data.InvalidParameters}`);
+    }
+
+    const pathToValue: any = {};
+    data.Parameters!.forEach(parameter => {
+        pathToValue[parameter.Name!] = parameter.Value;
+    });
+
+    return pathToValue;
+}
+
+export default async function getIntegrationTransformConfig(): Promise<{
+    integrationTransformUrl: string;
+    integrationTransformAwsRegion: string;
+}> {
+    const integrationTransformUrlPath = getParamPath(
+        'INTEGRATION_TRANSFORM_PATH',
+        `/fhir-service/integration-transform/${STAGE}/url`,
+    );
+    const integrationTransformAwsRegionPath = getParamPath(
+        'INTEGRATION_TRANSFORM_AWS_REGION_PATH',
+        `/fhir-service/integration-transform/${STAGE}/awsRegion`,
```

Comment:
What's the value of `STAGE` when running sls offline ?

Reply:
By default `stage` is set to be `dev`. But it's configurable, you can run `sls offine start --stage <STAGE>`